code/game/objects/machinery/cryopod.dm

```diff
@@ -172,34 +172,29 @@
 
 		visible_message(span_notice("[src] beeps happily as it disgorges the desired objects."))
 
+		var/list/da_list
```

Comment:
cringe var name

Reply:
Changed the var name.
Also, I figured having multiple big lists would be better when they get really full of crap. Now that's less of a problem with roomba refills, I think.